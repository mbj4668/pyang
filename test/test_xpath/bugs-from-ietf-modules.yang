module bugs-from-ietf-modules {
  namespace urn:example:bugs;
  prefix i;

  grouping common-session-parms {
    container x;
  }

  // bug from RFC 6470
  notification netconf-confirmed-commit {
    uses common-session-parms {
      when "../confirm-event != 'timeout'";
    }

    leaf confirm-event {
      type string;
    }
  }

  container snmp {
    container target {
    }
  }

  // bug from RFC 7407
  augment /snmp/target {
    when "v1 or v2c";
    leaf mms {
      type string;
    }
  }
}
