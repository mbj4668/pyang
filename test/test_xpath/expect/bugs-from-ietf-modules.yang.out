bugs-from-ietf-modules.yang:12: warning: XPATH_NODE_NOT_FOUND2
bugs-from-ietf-modules.yang:27: warning: XPATH_NODE_NOT_FOUND1
bugs-from-ietf-modules.yang:27: warning: XPATH_NODE_NOT_FOUND1
